Code/Mantid/Framework/Algorithms/inc/MantidAlgorithms/MonitorEfficiencyCorUser.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MANTID_ALGORITHMS_MONITOREFFICIENCYCORUSER_H_
+#define MANTID_ALGORITHMS_MONITOREFFICIENCYCORUSER_H_
+
+#include "MantidKernel/System.h"
+#include "MantidAPI/Algorithm.h"
+
+namespace Mantid
+{
+  namespace Algorithms
+  {
+
+class DLLExport MonitorEfficiencyCorUser : public API::Algorithm
```

Comment:
Would be great if you could apply clang-format to your source files. That keeps all our styles consistent. There are various plugins for development ides. See here http://clang.llvm.org/docs/ClangFormat.html